tzlocal/tz_windows.go

```diff
@@ -29,6 +29,23 @@ func LocalTZ() (string, error) {
 	if name, ok := WinTZtoIANA[winTZname]; ok {
 		return name, nil
 	}
+	
+	// If the timezone is not found in the WinTZtoIANA map, check if the timezone string contains "_dstoff",
+	// which indicates that "Daylight Saving Time" adjustments for the timezone are disabled.
+	// Remove the "_dstoff" part from the timezone string and check in the map again.
+	// Refer: https://learn.microsoft.com/en-us/windows-server/administration/windows-commands/tzutil#syntax
+	index := strings.Index(winTZname, "_dstoff")
```

Comment:
This can leverage HasSuffix and TrimSuffix

```go
if strings.HasSuffix(winTZname, "_dstoff") {
		// Remove "_dstoff" from the timezone string
		winTZname = strings.TrimSuffix(winTZname, "_dstoff")

		// Check the updated timezone string in the WinTZtoIANA map.
		if name, ok := tzlocal.WinTZtoIANA[winTZname]; ok {
			return name, nil
		}
	}
```

Reply:
If you simply strip the _dstoff and use the regular time zone mapping, you may end up with a time zone that inaccurately reflects the current time, which is something I need to verify before adding this change.

Reply:
`Pacific Standard Time` maps to `America/Los_Angeles`  

`Pacific Standard Time_dstoff` does not map to `America/Los_Angeles` all year round, as `America/Los_Angeles` includes Daylight Saving Time. Instead, it should map to a time zone like `Etc/GMT-8` that maintains a consistent UTC-8 offset year-round.

Reply:
can we implement something similar to this?

https://github.com/regebro/tzlocal/blob/52e53725f86fd74e551965de4e3a9f3aee826c8d/tzlocal/win32.py#L87C5-L96C28

Reply:
Yes, that would be a solution to the current problem, but I need to find a spare spot of time to work on it first, unless you or someone else wants to do it first.